test/test_v2/test_server.c

```diff
@@ -31,6 +31,10 @@
 
 int my_server_id = 0;
 
+// Pipes for dynamic side channel
+int child_write_pipe[PMIXT_MAX_LOCAL_PROCS][2];
+int server_write_pipe[PMIXT_MAX_LOCAL_PROCS][2];
+
```

Comment:
As this PR doesn't leverage this, let's remove it from here and PR when relevant.
This simplifies the review to me.
```suggestion
```

Reply:
Ok